fast5_research/fast5.py

```diff
@@ -119,8 +118,26 @@ def New(cls, fname, read='a', tracking_id={}, context_tags={}, channel_id={}):
                 'channel_id does not contain required fields: {},\ngot {}.'.format(req_fields.keys(), channel_id.keys())
             )
         channel_id = _type_meta(channel_id, req_fields)
+       
+        # tracking_id: spec. says this group should be present as string, says
+        #   nothing about required keys, but certain software require the
+        #   following minimal set
+        req_fields = {
+            # guppy
+            'exp_start_time': np.dtype('S20'),  # e.g. '1970-01-01T00:00:00Z'
+            'run_id': np.dtype('S32'),          #      '118ea414303a46d892603141b9cbd7b0'  
+            'flow_cell_id': np.dtype('S8'),     #      'FAH12345'
+            # ...add others
```

Comment:
shall we make that a TODO?

Reply:
The comment was really to suggest when others are added we should comment why (i.e. which program requires them).